assets/src/edit-story/components/colorPicker/pointer.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+const Pointer = styled.div`
+  width: ${({ size }) => `${size}px`};
+  height: ${({ size }) => `${size}px`};
+  background: radial-gradient(
+    circle at center ${({ size }) => `${size / 2}px`},
+    transparent,
+    transparent 4px,
+    #fff 4px
+  );
```

Comment:
Can this be a simple border?

Reply:
Also can we add the currently picked color as a background to the pointers?
![Screen Shot 2020-02-24 at 10 42 28 AM](https://user-images.githubusercontent.com/591655/75181889-0ee32e80-56f4-11ea-8dbc-bb7052072ea7.png)

Reply:
Added now, but the background color thing looks a bit odd at first glance:

![Screenshot 2020-02-25 at 14 34 14](https://user-images.githubusercontent.com/841956/75252032-1ed63d80-57dc-11ea-9ae1-d43fd26b359f.png)

Reply:
Ah yes, that doesn't work in the opacity thing - you probably need to add a white layer behind it too, otherwise the opacities stack (essentially doubling up).